config/stack_noaa.yaml

```diff
@@ -4,6 +4,7 @@ cmake:
 
 jpeg:
   build: NO
+  shared: NO
```

Comment:
If `build: NO` is the only difference between `stack_noaa.yaml` and `stack_ncar.yaml`, do you see any harm in turning jpeg to YES in `stack_noaa.yaml`?
I don't. May be @kgerheiser has some thoughts.

Reply:
I would prefer building it per default on NOAA platforms, too. In that case we could ditch the stack_ncar.yaml file again.

Reply:
I would in that case suggest just build jpeg on NOAA platforms and ditch `stack_ncar.yaml`